source/reference/minio-mc-admin/mc-admin-user-svcacct.rst

```diff
@@ -43,369 +44,37 @@ The :mc:`mc admin user svcacct` command has the following subcommands:
-
-   .. mc-cmd:: ALIAS
-      :required:
-
-      The :ref:`alias <alias>` of the MinIO deployment.
-
-   .. mc-cmd:: SERVICEACCOUNT
-      :required:
-
-      The access key for the access keys to enable.
-
-.. mc-cmd:: disable
-   :fullpath:
-
-   Disables a access keys for the specified user. Applications can only authenticate using enabled access keys. 
-
-   .. tab-set::
-
-      .. tab-item:: EXAMPLE
-
-         The following command disables the specified access keys:
-
-         .. code-block:: shell
-            :class: copyable
-
-            mc admin user svcacct disable myminio myuserserviceaccount
-
-      .. tab-item:: SYNTAX
-
-         The command has the following syntax:
-
-         .. code-block:: shell
-            :class: copyable
-
-            mc [GLOBALFLAGS] admin user svcacct disable  \
-                                        ALIAS            \
-                                        SERVICEACCOUNT
-
-   .. mc-cmd:: ALIAS
-      :required:
-
-      The :ref:`alias <alias>` of the MinIO deployment.
-
-   .. mc-cmd:: SERVICEACCOUNT
-      :required:
-
-      The access key for the access keys to disable.
-
-Global Flags
-~~~~~~~~~~~~
-
-.. include:: /includes/common-minio-mc.rst
-   :start-after: start-minio-mc-globals
-   :end-before: end-minio-mc-globals
\ No newline at end of file
+   * - :mc:`~mc admin user svcacct add`
+     - .. include:: /reference/minio-mc-admin/mc-admin-user-svcacct-add.rst
+          :start-after: start-mc-admin-svcacct-add-desc
+          :end-before: end-mc-admin-svcacct-add-desc
+
+   * - :mc:`~mc admin user svcacct disable`
+     - .. include:: /reference/minio-mc-admin/mc-admin-user-svcacct-disable.rst
+          :start-after: start-mc-admin-svcacct-disable-desc
+          :end-before: end-mc-admin-svcacct-disable-desc
+
+   * - :mc:`~mc admin user svcacct edit`
+     - .. include:: /reference/minio-mc-admin/mc-admin-user-svcacct-edit.rst
+          :start-after: start-mc-admin-svcacct-edit-desc
+          :end-before: end-mc-admin-svcacct-edit-desc
+
+   * - :mc:`~mc admin user svcacct enable`
+     - .. include:: /reference/minio-mc-admin/mc-admin-user-svcacct-enable.rst
+          :start-after: start-mc-admin-svcacct-enable-desc
+          :end-before: end-mc-admin-svcacct-enable-desc
+
+   * - :mc:`~mc admin user svcacct info`
+     - .. include:: /reference/minio-mc-admin/mc-admin-user-svcacct-info.rst
+          :start-after: start-mc-admin-svcacct-info-desc
+          :end-before: end-mc-admin-svcacct-info-desc
+
+   * - :mc:`~mc admin user svcacct list`
+     - .. include:: /reference/minio-mc-admin/mc-admin-user-svcacct-list.rst
+          :start-after: start-mc-admin-svcacct-list-desc
+          :end-before: end-mc-admin-svcacct-list-desc
+
+   * - :mc:`~mc admin user svcacct remove`
+     - .. include:: /reference/minio-mc-admin/mc-admin-user-svcacct-remove.rst
+          :start-after: start-mc-admin-svcacct-remove-desc
+          :end-before: end-mc-admin-svcacct-remove-desc
```

Comment:
AH. That's why.

@feorlen need a `.. toctree` directive here. I'd bet there are "orphaned document" or "not linked" warnings in your build output.

Reply:
Weird, I don't get a warning about that. I'll have a look.